safe-global/safe-wallet-web
# fix: Disable transaction buttons until sdk is initialized

## What it solves

Resolves #4404

This should make our e2e tests more robust

## How this PR fixes it

- Adds a check for the `sdk` inside `CheckWallet` which will disable all buttons it wraps until the SDK is initialized

## How to test it

1. Create and queue a transaction
2. Go to the queue
3. Reload the app
4. Observe the Confirm button is disabled until the SDK is ready

## Checklist
